### PR TITLE
Add Subtract<> function

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: functions/subtract.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    svfb_.run(blade);
+  }
+  int calculate(BladeBase* blade) {
+    return (svfa_.calculate(blade) - svfb_.calculate(blade));
+  }
+ private:
+  PONUA SVFA svfa_;
+  PONUA SVFB svfb_;
+};
+
```
Comment (profezzorn) [inline functions/subtract.h:39]: This won't work because it's missing the selection code.

Comment (fdarosa2663) [reply functions/subtract.h:39]: Is this what is missing, wasn't sure how the SVF was working
https://github.com/profezzorn/ProffieOS/blob/master/functions/sum.h#L40

Comment (profezzorn) [reply functions/subtract.h:39]: Basically yes, but you're probably better off using the code from the Mult<> function. Sum<> has some extra code to allow arbitrary number of arguments, which we probably don't want/need for Subtract<>.

File: functions/subtract.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#ifndef FUNCTIONS_SUBTRACT_H
+#define FUNCTIONS_SUBTRACT_H
+
+// Usage: SUBTRACT<A, B>
```
[the remark this answers is not in the document]
Comment (fdarosa2663) [reply functions/subtract.h]: fixed

File: functions/subtract.h [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+// Subtracts B from A (A - B)
+// A, B: INTEGER
```
[the remark this answers is not in the document]
Comment (fdarosa2663) [reply functions/subtract.h]: fixed

File: functions/subtract.h [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+// return value: INTEGER
```
[the remark this answers is not in the document]
Comment (fdarosa2663) [reply functions/subtract.h]: fixed

File: functions/subtract.h [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+template<class SVFA, class SVFB>
+class SubtractSVF {
+ public:
+  void run(BladeBase* blade) {
+    svfa_.run(blade);
+    svfb_.run(blade);
+  }
+  int calculate(BladeBase* blade) {
+    return (svfa_.calculate(blade) - svfb_.calculate(blade));
```
Comment (profezzorn) [inline functions/subtract.h:33]: remove extra parenthesis

Comment (fdarosa2663) [reply functions/subtract.h]: removed

**Pull request merged**